neo/Wallets/Wallet.py

```diff
@@ -67,9 +67,9 @@ def __init__(self, path, passwordKey, create):
         """
 
         Args:
-            path: (str) A path indicating where to create or open the wallet
-            passwordKey: (str) A password to use in creating or opening the wallet
-            create: (bool) Whether to create the wallet or simply open
+            path: (str) A path indicating where to create or open the wallet.
```

Comment:
The data type comes before the colon (so `path (str):`)

Reply:
/me looks @localhuman ;)
I didn't write those comments, but I can of course update them

Reply:
Plz 👍

Reply:
will do